tr.org.pardus.pen.yaml

```diff
@@ -0,0 +1,23 @@
+app-id: tr.org.pardus.pen
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+command: /app/usr/bin/pardus-pen
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
if it supports wayland then it's `fallback-x11`. Otherwise remove the wayland line.